.github/workflows/deploy-alpha.yml

```diff
@@ -0,0 +1,33 @@
+name: deploy-alpha
+
+concurrency: alpha
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  deployment:
+    if: github.event.pull_request.merged == true
```

Comment:
I think this will return false except when the release branch will be merged in the master no ?

Reply:
When a pull request is close (for what ever reason) and the branch name matches the glob and the pull request was merged this will run :)
The merge check is there because lerna also makes some commits.... which I'll need to look into to make sure it has access